src/modules/builder/configuration/AddLabelsStep/AddLabelForm.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { KeepScale } from 'react-zoom-pan-pinch';
+
+import { CloseRounded } from '@mui/icons-material';
+import { Box, Button, IconButton, Stack, TextField } from '@mui/material';
+
+import { useAppTranslation } from '@/config/i18n';
+import { APP } from '@/langs/constants';
+
+type Props = {
+  formPosition: { y: number; x: number };
```

Comment:
I was wondering what was "form" but it's actually the thing you use to input label. I think you can call it `position` when it's in its own component.
```suggestion
  position: Position;
```